module/token.js

```diff
@@ -0,0 +1,207 @@
+import { NUMENERA } from './config.js';
+
+export function cypherToken() {
+    // Here we monkey-patch in a bunch of crap because I can see no better way to have custom token behavior
```

Comment:
Sadly, if we could set an entityClass property on Token like we do with Actor or Item, that would be sweet, but in the meantime this seems the only way to go.

Reply:
That's rough, but on the one hand I'm glad that I didn't miss some obvious, superior solution.